src/Jackett/Indexers/nCore.cs

```diff
@@ -0,0 +1,214 @@
+                UseCookies = true,
+            };
+            client = new HttpClient(handler);
+        }
+
+
+        HttpRequestMessage CreateHttpRequest(string url)
+        {
+            var message = new HttpRequestMessage();
+            message.Method = HttpMethod.Get;
+            message.RequestUri = new Uri(url);
+            message.Headers.UserAgent.ParseAdd(BrowserUtil.ChromeUserAgent);
+            return message;
+        }
+
+        public Task<ConfigurationData> GetConfigurationForSetup()
+        {
+            var config = this.ConfigData == null ? new ConfigurationDatanCore() : new ConfigurationDatanCore(this.ConfigData);
+            return Task.FromResult<ConfigurationData>(config);
+        }
+
+        public async Task ApplyConfiguration(JToken configJson)
+        {
+            var config = new ConfigurationDatanCore();
+            config.LoadValuesFromJson(configJson);
+
+            if (config.Hungarian.Value == false && config.English.Value == false)
+                throw new ExceptionWithConfigData("Please select atleast one language.", (ConfigurationData)config);
+
+            var startMessage = CreateHttpRequest(LoginUrl);
+            var results = await (await client.SendAsync(startMessage)).Content.ReadAsStringAsync();
+
+
+            var pairs = new Dictionary<string, string> {
+				{ "nev", config.Username.Value },
+				{ "pass", config.Password.Value }
+			};
+
+            var content = new FormUrlEncodedContent(pairs);
+
+            var loginRequest = CreateHttpRequest(LoginUrl);
+            loginRequest.Method = HttpMethod.Post;
+            loginRequest.Content = content;
+            loginRequest.Headers.Referrer = new Uri(LoggedInUrl);
+
+            var response = await client.SendAsync(loginRequest);
+            var responseContent = await response.Content.ReadAsStringAsync();
+
+            if (!responseContent.Contains("Felhasználó"))
+            {
+                var errorMessage = "Couldn't login";
+                throw new ExceptionWithConfigData(errorMessage, (ConfigurationData)config);
+            }
+            else
+            {
+                var configSaveData = new JObject();
+                cookies.DumpToJson(SiteLink, configSaveData);
+                cookies.DumpConfigToJson(config, configSaveData);
+                SaveConfig(configSaveData);
+                IsConfigured = true;
+            }
+        }
+
+        async Task<ReleaseInfo[]> PerformQuery(TorznabQuery query, Uri baseUrl)
+        {
+            List<string> searchurls = new List<string>();
+            var searchString = query.SanitizedSearchTerm + " " + query.GetEpisodeSearchString();
+            searchurls.Add(String.Format(SearchUrl, 1, HttpUtility.UrlEncode(searchString)));
+            searchurls = getSearchUrls(searchurls, searchString).Result;
+
+            List<ReleaseInfo> releases = new List<ReleaseInfo>();
+            foreach (string url in searchurls)
```

Comment:
We shouldn't get more than the first page unless we have a search term, it just adds excess load.  There is some upcoming adaptive updating stuff which will help ensure items are not missed.